index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./main.css" />
+        <title>Document</title>
+    </head>
+    <body>
+        <div class="container">
+            <div class="choice-item choice-1">
+                <div class="head">
+                    <div class="my-img"><img src="./img/ocean.png" alt=""></div>
+                    <p class="title">OCEAN</p>
+                </div>
+                <div class="price">$399</div>
+                <div class="subtitle">
+                    <p>Subtitle goes here</p>
+                    <p>"Subtitle second line goes here lorem ipsum"</p>
+                </div>
+                <ul>
+                    <li class="list-item">Drop out ramen hustle</li>
+                    <li class="list-item">Coworking viral landing</li>
+                    <li class="list-item">Crush revenue traction</li>
+                    <li class="list-item">User base minimum viable</li>
+                    <li class="list-item">Lorem ipsum dolor</li>
+                </ul>
+                <button>BUY TOUR</button>
+            </div>
+            <div class="choice-item choice-2">
+                <div class="head">
+                    <div class="my-img"><img src="./img/mountain.png" alt=""></div>
+                    <p class="title">MOUNTAINS</p>
+                </div>
+                <div class="price">$499</div>
+                <div class="subtitle">
+                    <p>Subtitle goes here</p>
+                    <p>"Subtitle second line goes here lorem ipsum"</p>
+                </div>
+                <ul>
+                    <li class="list-item-1">Drop out ramen hustle</li>

```

Comment:
mới sửa ul li đầu tiên thôi e, đây là trong trường hợp chỉ có duy nhất 3 cái cục biết trước như này, nên set css ul li cũng được (hiện tại a chưa thấy dùng đến class list-item), nhưng nếu trong dự án sẽ k beíet trước được toàn bộ UI, set thẳng thẻ như thế sẽ ảnh hưởng.